catalogue/development.yml

```diff
@@ -39,17 +39,29 @@ choices:
       - rpm:kate
       - flatpak:org.kde.kate
 
-  - name: Turtle
-    provider: Philipp Unger
-    description: |
-      Turtle provides a command line tool, which opens a dialog for some git actions, i.e. commit, log, etc.
-      It also provides a plugin for Nautilus (and other file managers), which adds emblems and context menu entries. 
-    actions: flatpak:de.philippun1.turtle
+  - name: virt-manager
+    provider: virt-manager
+    description: Desktop tool for managing virtual machines via libvirt.
+    actions: rpm:virt-manager
 
-  - name: Pods
-    provider: Marcus Behrendt
-    description: Pods is a frontend for podman. It uses libadwaita for its user interface. 
-    actions: flatpak:com.github.marhkb.Pods
+  - name: Tailscale
+    provider: Tailscale
+    description: Tailscale is a mesh VPN service that streamlines connecting devices and services securely across different networks.
+    actions: enable_yum_repo:https://pkgs.tailscale.com/stable/fedora/tailscale.repo;rpm:tailscale;shell:systemctl enable tailscaled
+
+  - name: Docker
+    provider: Docker
+    description: Software that uses virtualization to create and manage containers.
```

Comment:
maybe mention in a note: Podman incomes with Ultramarine and supports Docker containers